### PR TITLE
BaSP-MR-229: Login with firebase authentication 

### PR DESCRIPTION
**Pull request by maurojjzz**
Create new component called Auth which inside has Login component. On it was implemented the aunthentication using firebase.
A little modifications on global buttons component and styles for Login Form.

### CODE REVIEW

**Comment by maurojjzz** [comment]
[FORM Login](https://trello.com/c/VJD1kcMs/229-form-login)

File: src/redux/auth/thunks.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ export const login = (credentials) => {
       } = await firebaseResponse.user.getIdTokenResult();
       return dispatch(loginSuccess({ role, token }));
     } catch (error) {
+      console.log(error);
```
Comment (CristianLotorto) [inline src/redux/auth/thunks.js:31]: you should replace this console.log

**Review by YhanahiRosenthal — APPROVED** [review]
Good job!

**Review by CristianLotorto — APPROVED** [review]
Look the comment! Great work!

**Pull request merged**